src/__mocks__/models/proposal.ts

```diff
@@ -0,0 +1,7 @@
+import { Proposal } from 'models';
+import { IProposalApiResponse } from 'types/apiResponses/proposal';
+import proposalResponse from '../api/proposal.json';
+
+const proposals = proposalResponse.result.map(p => new Proposal(p as IProposalApiResponse));
```

Comment:
So following our quick discussion yesterday, I'd propose to move to using types rather than models, this way each type isn't tied to specific requests

Reply:
Whether it is a a type or a model they will represent two things 1) the returned response 2) the shape of data in the app. 

I think models give us an advantage over returning and object. The model can return memoized values via getters, can easily be extended, and doesn't need an extra type defined in addition to the object. 

We don't have to take all the values from the response, we can take just the ones we need which is what I understood your point to be from the discussion

Reply:
Another advantage is shown here. Depending on what we are testing we may want the raw API Response or we may want the santized and formatted response that we will be using in the application. Following this pattern will give us clean mocks with clear purposes and definitions

Reply:
Alternatively if we prefer to return a plain object, we can write a function that would be in place of the class to format the data and could also be extended but wouldn't have the added benefit of memoized getters. This format function could be used to follow the mocking pattern I mentioned in the previous comment